src/droplet.F

```diff
@@ -1932,15 +1941,83 @@ subroutine droplet_diag(rtime,nrec,zh,zf,pdata,pdata_locind,dpten)
       end if
 #endif
 
+
+
+      !!!!!!! HISTOGRAMS  !!!!!!!!
+
+      ! One last type of droplet stat is the histogram
+
+      bins_radius = 0.0
+      hist_radius = 0.0
+      hist_radius10 = 0.0
+
+      !First set up the bin centers
+      rmin = 1.0e-8
+      rmax = 1.0e-3
+      rmin10 = log10(rmin)
+      rmax10 = log10(rmax)
+
+      dr = (rmax10-rmin10)/(nbins-1)
+
+      bins_radius(1) = rmin10-dr
+      do i=1,nbins-1
+         bins_radius(i+1) = bins_radius(i) + dr
+      end do 
+
+      !Now that the bin centers are defined, loop through particles
+      do np=1,nparcelsLocalActive
```

Comment:
either need an OpenACC directive for this loop or update`pdata` on the host first.